calibr8/utils.py

```diff
@@ -166,29 +168,33 @@ def plot_t_band(ax, independent, mu, scale, df, *, residual_type: typing.Optiona
     residual_type : str, optional
         One of { None, "absolute", "relative" }.
         Specifies if bands are for no, absolute or relative residuals.
-
     Returns
     -------
     artists : list of matplotlib.Artist
         the created artists (1x Line2D, 6x PolyCollection (alternating plot & legend))
     """
+    import warnings
+    warnings.warn(
+            " `plot_t_band` is substituted by a more genereal `plot_continuous_band` function."
```

Comment:
```suggestion
            "`plot_t_band` is substituted by a more general `plot_continuous_band` function."
```
The same typo is probably in the other warning too. Since you're probably gonna edit this locally, I won't bother making the other comment on my phone..